<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartObjectPatternDialog</class>
 <widget class="QDialog" name="SmartObjectPatternDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>State Pattern</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Pattern matches if at least one of these sub patterns match:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>m_wndList</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="m_wndList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnNew">
       <property name="text">
        <string>&amp;New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnEdit">
       <property name="text">
        <string>&amp;Edit...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnDelete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SmartObjectPatternDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SmartObjectPatternDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
